### PR TITLE
Mandrel 21.3.2.0 and 22.1.0.0

### PR DESCRIPTION
**Pull request by Karm**
Release:
*	21.3.2.0-Final-java11/
*	21.3.2.0-Final-java17/
*	22.1.0.0-Final-java11/
*	22.1.0.0-Final-java17/

+ reshuffling tags to reflect the latest.

### CODE REVIEW

**Comment by Karm** [comment]
Hello @cescoffier, I don't understand the message:
```
2022-04-28 07:05:03,233 cli.py:477        ERROR Module 'mandrel' with version '21.3.1.1-Final-java11' already exists in module registry
```

What does it mean and how do I get rid of it?

File: modules/mandrel/21.3.2.0-Final-java11/module.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+schema_version: 1
+name: mandrel
+version: &version "21.3.1.1-Final-java11"
```
Comment (zakkak) [inline modules/mandrel/21.3.2.0-Final-java11/module.yaml:3]:
```suggestion
version: &version "21.3.2.0-Final-java11"
```

File: modules/mandrel/21.3.2.0-Final-java11/module.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+labels:
+  - name: mandrel-archive-filename
+    value: &filename mandrel-java11-linux-amd64-21.3.1.1-Final.tar.gz
```
Comment (zakkak) [inline modules/mandrel/21.3.2.0-Final-java11/module.yaml:7]:
```suggestion
    value: &filename mandrel-java11-linux-amd64-21.3.2.0-Final.tar.gz
```

File: modules/mandrel/21.3.2.0-Final-java11/module.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  - name: mandrel-archive-url
+    value: &url https://github.com/graalvm/mandrel/releases/download/mandrel-21.3.1.1-Final/mandrel-java11-linux-amd64-21.3.1.1-Final.tar.gz
```
Comment (zakkak) [inline modules/mandrel/21.3.2.0-Final-java11/module.yaml:9]:
```suggestion
    value: &url https://github.com/graalvm/mandrel/releases/download/mandrel-21.3.2.0-Final/mandrel-java11-linux-amd64-21.3.2.0-Final.tar.gz
```

File: modules/mandrel/22.1.0.0-Final-java11/module.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+schema_version: 1
+name: mandrel
+version: &version "22.0.0.2-Final-java11"
+
+labels:
+  - name: mandrel-archive-filename
+    value: &filename mandrel-java11-linux-amd64-22.0.0.2-Final.tar.gz
+  - name: mandrel-archive-url
+    value: &url https://github.com/graalvm/mandrel/releases/download/mandrel-22.0.0.2-Final/mandrel-java11-linux-amd64-22.0.0.2-Final.tar.gz
```
Comment (zakkak) [inline modules/mandrel/22.1.0.0-Final-java11/module.yaml:9]:
```suggestion
    value: &url https://github.com/graalvm/mandrel/releases/download/mandrel-22.1.0.0-Final/mandrel-java11-linux-amd64-22.1.0.0-Final.tar.gz
```

**Comment by Karm** [comment]
> Thanks for preparing the PR @Karm , it looks like some versions were not updated after copying the corresponding files :)

:man_facepalming:

**Review by cescoffier — APPROVED** [review]

**Comment by Karm** [comment]
THX @cescoffier 
Would it be possible to kick off the build so as we can have it on quay soon, please? I am not sure whether we didn't miss the weekly build window...

**Comment by cescoffier** [comment]
@Karm I'm just back from PTO and I need to add a few new graalvm images today. Once merge, I will trigger the deployment

**Comment by cescoffier** [comment]
Done!

**Pull request merged**